tests/e2e/paths/search/granules/granules.spec.js

```diff
@@ -0,0 +1,1244 @@
+import timelineCollectionsBody from './__mocks__/timeline/collections.body.json'
+import timelineCollectionsHeaders from './__mocks__/timeline/collections.headers.json'
+import timelineGranulesBody from './__mocks__/timeline/granules.body.json'
+import timelineGraphQlBody from './__mocks__/timeline/graphql.body.json'
+import timelineGraphQlHeaders from './__mocks__/timeline/graphql.headers.json'
+import timelineTimelineBody from './__mocks__/timeline/timeline.body.json'
+import timelineTimelineHeaders from './__mocks__/timeline/timeline.headers.json'
+import graphQlHeaders from './__mocks__/common/graphql.headers.json'
+
+import { login } from '../../../../support/login'
+
+const defaultCmrPageSize = 20
+
+const testResultsSize = async (page, cmrHits) => {
+  const expectedSize = Math.min(defaultCmrPageSize, cmrHits)
+
+  const metaText = await page.getByTestId('panel-group_granule-results')
+    .getByTestId('panel-group-header__heading-meta-text')
+    .textContent()
+  expect(metaText).toBe(`Showing ${expectedSize} of ${commafy(cmrHits)} matching ${pluralize('granule', cmrHits)}`)
+}
+
+test.describe('Path /search/granules', () => {
+  test.beforeEach(async ({ page }) => {
+    const fakeNow = new Date('2021-06-01').valueOf()
+
+    await page.addInitScript(`{
+      Date = class extends Date {
+        constructor(...args) {
+          if (args.length === 0) {
+            super(${fakeNow});
+          } else {
+            super(...args);
+          }
+        }
+      }
+
+      const __DateNowOffset = ${fakeNow} - Date.now();
+      const __DateNow = Date.now;
+      Date.now = () => __DateNow() + __DateNowOffset;
+    }`)
+  })
+
+  test.describe('When the path is loaded with only the collectionId', () => {
+    test('loads correctly', async ({ page }) => {
+      const conceptId = 'C1214470488-ASF'
+      const cmrHits = 1059170
+
+      await page.route('**/search/granules.json', (route) => {
+        const request = route.request()
+        const body = request.postData()
+        expect(body).toBe('echo_collection_id=C1214470488-ASF&page_num=1&page_size=20')
+        route.fulfill({
+          body: JSON.stringify(noParamsGranulesBody),
+          headers: {
+            ...commonGranulesHeaders,
+            'access-control-expose-headers': 'cmr-hits',
+            'cmr-hits': cmrHits
+          }
+        })
+      })
+
+      await page.route('**/search/granules/timeline', (route) => {
+        const request = route.request()
+        const body = request.postData()
+        if (body) {
+          expect(body).toBe('end_date=2023-12-01T00:00:00.000Z&interval=day&start_date=2018-12-01T00:00:00.000Z&concept_id[]=C1214470488-ASF')
+        }
+
+        route.fulfill({
+          body: JSON.stringify(noParamsTimelineBody),
+          headers: noParamsTimelineHeaders
+        })
+      })
+
+      await page.route('**/api', (route) => {
+        const request = route.request()
```

Comment:
I don't think we need to do the stringify and unstringify stuff here I think a few other spots correct me if Im wrong

Reply:
Simplified